scripts/jenkins/ardana/ansible/init.yml

```diff
@@ -10,6 +10,15 @@
     ARDANA_INIT_AUTO: 1
 
   tasks:
+  - name: Install growpart
+    become: yes
+    shell: zypper -n in http://download.opensuse.org/update/leap/42.1/oss/noarch/growpart-0.29-7.1.noarch.rpm
```

Comment:
- ```growpart``` is in the pubcloud module. Can we use it from there?
- you can use the ```zypper``` ansible module instead of using the shell (see http://docs.ansible.com/ansible/latest/zypper_module.html )

Reply:
the version that is available is broken (has a syntax error). thats why I'm specifically using one that has been empirically tested to work.

Reply:
what we can do instead is upload a new image that has a fixed growpart already embedded. working on that.